File: Beecrowd/Beginner/1000 - 1099/1061 - Event Time/C/1061_EventTime.c
#include <stdio.h>
#include <string.h>

int main() {
    int day1, hour1, minute1, second1;
    int day2, hour2, minute2, second2;
    int day, hour, minute, second;
    char word[10];

    scanf("%s %d", word, &day1);
    scanf("%d %s %d %s %d", &hour1, word, &minute1, word, &second1);
    scanf("%s %d", word, &day2);
    scanf("%d %s %d %s %d", &hour2, word, &minute2, word, &second2);

    second = second2 - second1;
    minute = minute2 - minute1;
    hour = hour2 - hour1;
    day = day2 - day1;

    if (second < 0) {
        second += 60;
        minute--;
    }

    if (minute < 0) {
        minute += 60;
        hour--;
    }

    if (hour < 0) {
        hour += 24;
        day--;
    }

    printf("%d dia(s)\n", day);
    printf("%d hora(s)\n", hour);
    printf("%d minuto(s)\n", minute);
    printf("%d segundo(s)\n", second);

    return 0;
}

File: Beecrowd/1 - Beginner/1000 - 1099/1061 - Event Time/1061_EventTime.py
word = input().split()
day1 = int(word[1])
hour1, minute1, second1 = map(int, input().split()[0::2])
word = input().split()
day2 = int(word[1])
hour2, minute2, second2 = map(int, input().split()[0::2])

second = second2 - second1
minute = minute2 - minute1
hour = hour2 - hour1
day = day2 - day1

if second < 0:
    second += 60
    minute -= 1

if minute < 0:
    minute += 60
    hour -= 1

if hour < 0:
    hour += 24
    day -= 1

print(f"{day} dia(s)")
print(f"{hour} hora(s)")
print(f"{minute} minuto(s)")
print(f"{second} segundo(s)")
